src/test/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version = "1.0" encoding = "utf-8"?>
+<!DOCTYPE hibernate-configuration SYSTEM
+        "http://www.hibernate.org/dtd/hibernate-configuration-3.0.dtd">
+<hibernate-configuration>
+    <session-factory>
+        <property name="hibernate.dialect">
+            org.hibernate.dialect.H2Dialect
+        </property>
+
+        <property name="hibernate.connection.driver_class">
+            org.h2.Driver
+        </property>
+
+        <property name="hibernate.connection.url">
+            jdbc:h2:mem:postgres;DB_CLOSE_DELAY=-1;MVCC=true
```

Comment:
Adding `MVCC=true` fixed the failing tests locally for me.

This is enabling Multi-Version Concurrency Control for H2.

Find details on it here: https://stackoverflow.com/questions/4162557/timeout-error-trying-to-lock-table-in-h2 and here: http://www.h2database.com/html/advanced.html#mvcc

Reply:
@njd9419 I'm getting this error outside of the test environment now when I query for contributions/images, and this fix only works for h2

Reply:
Actually it's the references that's doing it:
>  Resolved [org.springframework.http.converter.HttpMessageNotWritableException: Could not write JSON: failed to lazily initialize a collection of role: com.monumental.models.Monument.references, could not initialize proxy - no Session; nested exception is com.fasterxml.jackson.databind.JsonMappingException: failed to lazily initialize a collection of role: com.monumental.models.Monument.references, could not initialize proxy - no Session (through reference chain: com.monumental.models.Monument["description"])]

Reply:
Never mind, I just need to add @JSONIgnore again